src/Network/Http/Adapter/Stream.php

```diff
@@ -305,4 +314,16 @@ public function contextOptions()
     {
         return $this->_contextOptions;
     }
+
+    /**
+     * Get the ssl context options
+     *
+     * Useful for debugging and testing ssl context creation.
+     *
+     * @return array
+     */
+    public function sslContextOptions()
```

Comment:
Couldn't contextOptions() just return a merge of the http + ssl context options. To the external user of this class they are no different.